### DISCUSSION TITLE
Run optimization iteratively

### DISCUSSION THREAD

**Discussion by zzzrbx** [optuna/optuna#5618] [Q&A]
### Don't use GitHub Issues to ask support questions.

Is it possible to run trials iteratively, for example:
for i in range(0, num_experiments):
    # do something. 
    # get run new trial. 
    # get results. 
    # do something else.

**Comment by nzw0301** [comment]
I'm unsure about the question, but the pattern looks similar to https://optuna.readthedocs.io/en/stable/tutorial/20_recipes/009_ask_and_tell.html.
